src/lib/Popup.svelte

```diff
@@ -21,6 +21,8 @@
   /** Define when to open the popup. If set to manual, you can open the popup programmatically by
    * setting the `open` attribute. */
   export let openOn: 'hover' | 'click' | 'dblclick' | 'contextmenu' | 'manual' = 'click';
+  /** Only open the popup if there's no feature from a higher layer covering this one. */
```

Comment:
Oh yeah, another question: should this even be a separate option, or just inherit the layer's `eventsIfTopMost`?

Reply:
I can imagine wanting different behaviours depending on the user action. For example if you still want to capture events from all layers to use in some other part of the app but only show the top level hover / click features.